docs/tutorial/window-customization.md

```diff
@@ -267,3 +267,15 @@ draggable areas.
 [overlay-css-env-vars]: https://github.com/WICG/window-controls-overlay/blob/main/explainer.md#css-environment-variables
 [overlay-javascript-apis]: https://github.com/WICG/window-controls-overlay/blob/main/explainer.md#javascript-apis
 [Window Controls Overlay API]: https://github.com/WICG/window-controls-overlay/blob/main/explainer.md
+
+## Panel windows _macOS_
+
+To achieve a floating panel that doesn't steal focus from a normal window,
+similar to how Spotlight or Alfred behave in macOS, use `type: "panel"` within `BrowserWindow`:
```

Comment:
On macOS, [Panels](https://developer.apple.com/design/human-interface-guidelines/panels)
are application windows that float above other windows without stealing focus (e.g. the
native Spotlight app). To make a BrowserWindow into a panel, set `type: "panel"` in its
constructor when initializing it.